renv.lock

```diff
@@ -4,7 +4,7 @@
     "Repositories": [
       {
         "Name": "CRAN",
-        "URL": "https://demo.rstudiopm.com/cran/latest"
+        "URL": "https://cloud.r-project.org"
```

Comment:
This ensures that we get mac binaries; I have no idea what impact this has on long term reproducibility